goosebit/settings/const.py

```diff
@@ -1,8 +1,11 @@
 from pathlib import Path
 
+import platformdirs
 from argon2 import PasswordHasher
 
 BASE_DIR = Path(__file__).resolve().parent.parent.parent
+CONFIG_DIR = platformdirs.site_config_path("gooseBit")
+DATA_DIR = platformdirs.site_data_path("gooseBit")
```

Comment:
```
>>> platformdirs.site_data_path("gooseBit")
PosixPath('/usr/share/ubuntu/gooseBit')
```
Better, but, I would expect:
```
/usr/share/goosebit
```

Also, the dev use case (as it was before) should still be covered. You don't want to run the app as `root`.

Reply:
Not sure how to cover the dev use case here.  No way to know if we're installed from pip vs in a dev environment, technically you can just run in dev by specifying the DB and migrations locations manually, but that's a bit annoying.

Reply:
The solution is to look in multiple places. Many applications do it that way.

Reply:
> The solution is to look in multiple places. Many applications do it that way.

We want to write to `DATA_DIR`, do we have a preference for location of that?

Reply:
For writing the migrations, only the dev use case applies, so `<project root>/goosebit/db/migrations/` seems fine to me. And for reading, there where they are now, in the package directory, is fine too. I mean, it is even Python code.

Reply:
> For writing the migrations, only the dev use case applies, so `<project root>/goosebit/db/migrations/` seems fine to me. And for reading, there where they are now, in the package directory, is fine too. I mean, it is even Python code.

Works for me.  And the user can already set the DB location in the config, so thats fine.  Don't need a DATA_DIR.

Reply:
Yes, same goes for `artifacts`. The current default is fine. When installed with pip, the default will not be writable by the application (unless run as root), so the user needs to configure the path. A little ugly, but it should work.